### PR TITLE
Fixes for vmware-esxi builder and any builders that the fuse scripts.

### PR DESCRIPTION
**Pull request by ThinGuy**
Changes to vmware-esxi to build ESXi 8.0b
Change any call that attempts `fusermount -u` to `fusermount -z -u`

### CODE REVIEW

**Comment by CodeBleu** [comment]
@ThinGuy I was looking around to see if any newer forks had any updates in them to fix the fuse stuff, and possibly Fedora38 OS as well...or any Fedora Server for that matter.  I then found your fork and was noticing you knocking stuff out.  Appreciate it! 

I'm assuming this will fix #126 ?

**Comment by ThinGuy** [comment]
> @ThinGuy I was looking around to see if any newer forks had any updates in them to fix the fuse stuff, and possibly Fedora38 OS as well...or any Fedora Server for that matter. I then found your fork and was noticing you knocking stuff out. Appreciate it!
> 
> I'm assuming this will fix #126 ?

I hope so! Currently, rhel 9.2 builds clean but the installation hits the 30-minute timeout.  Curtin runs without error, server is set up correctly, but it never tells MAAS that the installation is completed.

File: rhel9/README.md [hunk 1]
```diff
@@ -8,7 +8,14 @@ The Packer template in this directory creates a RHEL 9 AMD64 image for use with
 
 * A machine running Ubuntu 22.04+ with the ability to run KVM virtual machines.
 * qemu-utils, libnbd-bin, nbdkit and fuse2fs
-* [Packer](https://www.packer.io/intro/getting-started/install.html), v1.8.0 or newer
+* packer - from Hashicorp repository, v1.9.0 or newer
```
Comment (alexsander-souza) [inline rhel9/README.md:11]: keep the reference to Packer install page. It's easier to keep docs up to date if we rely on external resources.

File: ubuntu/Makefile [hunk 2]
```diff
@@ -8,8 +8,11 @@ SUDO ?= sudo
 all: custom-cloudimg.tar.gz
 
 check-deps:
-	dpkg -s libnbd-bin nbdkit packer fuse2fs cloud-image-utils ovmf > /dev/null
-
+	@if [ $(shell lsb_release -sr|cut -d. -f1) -le 20 ];then \
```
Comment (alexsander-souza) [inline ubuntu/Makefile:11]: see suggestion in the RHEL makefile above

File: vmware-esxi/Makefile [hunk 3]
```diff
@@ -10,7 +10,11 @@ VENV := .ve
 all: vmware-esxi.dd.gz
 
 check-deps:
-	dpkg -s libnbd-bin nbdkit packer fusefat > /dev/null
+	@if [ $(shell lsb_release -sr|cut -d. -f1) -le 20 ];then \
```
Comment (alexsander-souza) [inline vmware-esxi/Makefile:13]: see suggestion in the makefiles above

File: vmware-esxi/vmware-esxi.pkr.hcl [hunk 4]
```diff
@@ -25,9 +25,9 @@ source "qemu" "esxi" {
   headless         = true
   iso_checksum     = "none"
   iso_url          = var.vmware_esxi_iso_path
-  memory           = 4096
+  memory           = 8192
```
Comment (alexsander-souza) [inline vmware-esxi/vmware-esxi.pkr.hcl:28]: it this really necessary?

**Review by alexsander-souza — CHANGES_REQUESTED** [review]
awesome job, but it needs some tweaks

**Comment by alexsander-souza** [comment]
superseded by #137, #138, #139 and #140

**Pull request closed without merging**